### PR TITLE
#200: Namegiving Conventions->Naming Conventions & Fix typos in text …

### PR DESCRIPTION
**Pull request by gabriellsesam**
…and image

### CODE REVIEW

**Review by erik-leven — APPROVED** [review]

File: training/010_architecture_and_concepts/020_Beginner.rst [hunk 1]
```diff
@@ -444,22 +438,30 @@ pipe name i.e., “person-child-hr".
...
 An output pipe should have the same name as the name of the pipe
 generating the output pipe’s source dataset, only postfixed with
-“output” i.e., “person-child-hr-endpoint”.
+“endpoint” i.e., “person-child-hr-endpoint”.
 
 The following flow shows a typical Sesam flow with each pipe’s preferred
 name with an example:
 
+|
+
```
Comment (ga-hegsvold) [inline training/010_architecture_and_concepts/020_Beginner.rst:447]: Would have liked to see these two following figures tagged as "examples", but not sure how/if that is supported by RST.
Anyone found a sensible way to do that yet?

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]
A few comments and (mostly open) questions.

File: training/010_architecture_and_concepts/020_Beginner.rst [hunk 2]
```diff
@@ -361,20 +361,14 @@ JSON configuration of pipes and systems
...
-.. _namegiving-conventions-1-1:
+.. _naming-conventions-1-1:
 
-Namegiving conventions
-~~~~~~~~~~~~~~~~~~~~~~
-
-How (maybe a table) to give good names to the different parts in Sesam.
-
-Why this is *really* important
+Naming Conventions
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply training/010_architecture_and_concepts/020_Beginner.rst:366]: Hmm, good point. Personally I'd say capitalize for titles, but not for prepositions or articles (i.e. a/an/the, on, at, for...)

File: training/010_architecture_and_concepts/020_Beginner.rst [hunk 3]
```diff
@@ -408,9 +402,9 @@ easier in the long run.
 
 Input pipes should be named according to endpoint/table they connect to
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply training/010_architecture_and_concepts/020_Beginner.rst]: Fixed

File: training/010_architecture_and_concepts/020_Beginner.rst (hunk 1 above)
Comment (erik-leven) [reply training/010_architecture_and_concepts/020_Beginner.rst:447]: Not sure on how you wish that tag to be

File: training/010_architecture_and_concepts/020_Beginner.rst [hunk 4]
```diff
@@ -361,20 +361,14 @@ JSON configuration of pipes and systems
...
 
 When constructing an integration flow in Sesam the use of a standardized
-naming convention becomes essential. This becomes especially important
-when the integration project grows to more than a few pipes. A
-standardized naming convention helps you to easily structure your Sesam
-architecture such that:
+naming convention becomes essential as the project grows to more than a few pipes.
+A standardized naming convention helps you to easily structure your Sesam architecture such that:
 
 -  Localizing specific flows becomes easier.
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply training/010_architecture_and_concepts/020_Beginner.rst:373]: done

File: training/010_architecture_and_concepts/020_Beginner.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (ga-hegsvold) [reply training/010_architecture_and_concepts/020_Beginner.rst:366]: Yeah, I agree. Consistency is more important that which form we pick.
@gabriellsesam Any preferences on this? Maybe we should discuss it in the next meeting?

File: training/010_architecture_and_concepts/020_Beginner.rst (hunk 1 above)
Comment (ga-hegsvold) [reply training/010_architecture_and_concepts/020_Beginner.rst:451]: It's more a question of what RST supports and how our/a tool chain can recognize it and handle it properly.
`<broken-record>`I would not be surprised if we at some point in the near future will see benefits of being able to show all our examples (and figures) listed with proper numbers and titles.`</broken-record>`

Comment (ga-hegsvold) [reply training/010_architecture_and_concepts/020_Beginner.rst:451]: I'll resolve this for now, but I'm really keen to find out if this fesable with RST.

File: training/010_architecture_and_concepts/020_Beginner.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (gabriellsesam) [reply training/010_architecture_and_concepts/020_Beginner.rst:366]: I think it makes sense to only capitalize the first word

[the remark this answers is not in the document]
Comment (ga-hegsvold) [reply training/010_architecture_and_concepts/020_Beginner.rst:366]: It's also a bit easier to remember which words to capitalize if we only do it on the first word. 👴🏻

[the remark this answers is not in the document]
Comment (erik-leven) [reply training/010_architecture_and_concepts/020_Beginner.rst]: fixed

**Comment by gabriellsesam** [comment]
Warning, treated as error:
/home/travis/build/datanav/docs/training/030_dtl/020_Beginner.rst:145: WARNING: undefined label: namegiving-conventions-1-1 (if the link has no caption the label must precede a section header)

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]
I referenced this section in a previous commit:
training/030_dtl/020_Beginner.rst:146::ref:`namegiving-conventions-1-1`,
training/030_dtl/020_Beginner.rst:172::ref:`namegiving-conventions-1-1`

Could you please update these as well?

**Review by ga-hegsvold — APPROVED** [review]

**Pull request merged**